lib/redux/state/app_state.dart

```diff
@@ -95,7 +98,9 @@ class GlobalState {
   final Map<String, String> organizationsSlugByProjectSlug;
   final Map<String, Cursor> projectCursorsByOrganizationSlug;
   final Map<String, List<Project>> projectsByOrganizationSlug;
-  final bool projectsFetchedOnce;
+  final bool isLoading;
```

Comment:
Maybe we can get some context in the app names, as they represent the state of orgs and projects loading.

Reply:
Sorry not sure what you mean here.

Reply:
`isLoading` -> `isLoadingOrgsAndProjects`, as `isLoading` is too general for the whole app state.